app/models/asset_field_type.rb

```diff
@@ -52,6 +52,15 @@ def mapping
 
   private
 
+  def is_image?(mime_type)
+    ['image/jpeg', 'image/pjpeg', 'image/png','application/pdf' ,'image/x-png', 'image/gif'].include?(mime_type)
```

Comment:
`MimeMagic` gives you a method that allows you to infer what sort of type a mimetype is. Try: `MimeMagic.new(mime_type).mediatype == 'image'`. I'd also add a manual check for PDF. Either way, this `is_image?` method misses on some of our supported filetypes, like BMP.

Additionally, please follow the Ruby convention for predicate methods - this should be `image?` instead of `is_image?`

Edit: I see there are two methods that do similar things. This is definitely where a good name comes in handy - this should be named something more like `can_post_process?` or `has_thumb?`